en.lproj/Localizable.strings

```diff
@@ -3134,7 +3134,7 @@
 // MARK: Ca Phe Sua (Vietnamese Coffee) Page
 
 /* Vietnamese Coffee Introduction 1 */
-"VietnameseCoffee.Introduction.1" = "##Cà Phê## (coffee) was introduced to Vietnam in 1857 by a French priest. What makes Vietnamese ##Cà Phê## (coffee) special is the used of ##Sữa Đặc## (sweetened condensed milk), because fresh milk was not readily available at the time. The term ##Cà Phê Sữa## is a combination of the ##Cà Phê## (coffee) and ##Sữa Đặc## (sweetened condensed milk) to mean Vietnamese coffee. ##Cà Phê Sữa## (Vietnamese coffee) can be enjoyed hot or cold. The normal presentation of ##Cà Phê Sữa## (Vietnamese coffee) is you have the ##Sữa Đặc## (sweetened condensed milk) in a cup and the Phin filter on top that drips the ##Cà Phê## (coffee) over the ##Sữa Đặc## (sweetened condensed milk).";
+"VietnameseCoffee.Introduction.1" = "##Cà Phê## (coffee) was introduced to Vietnam in 1857 by a French priest. What makes Vietnamese ##Cà Phê## (coffee) special is the used of ##Sữa Đặc## (sweetened condensed milk), because fresh milk was not readily available at the time. The term ##Cà Phê Sữa## is a combination of the ##Cà Phê## (coffee) and ##Sữa Đặc## (sweetened condensed milk) to mean Vietnamese coffee. ##Cà Phê Sữa## (Vietnamese coffee) can be enjoyed hot or cold. The normal presentation of ##Cà Phê Sữa## (Vietnamese coffee) is you have the ##Sữa Đặc## (sweetened condensed milk) in a cup and the ##Phin Lọc## (Phin filter) on top that drips the ##Cà Phê## (coffee) over the ##Sữa Đặc## (sweetened condensed milk).";
```

Comment:
```suggestion
"VietnameseCoffee.Introduction.1" = "##Cà Phê## (coffee) was introduced to Vietnam in 1857 by a French priest. What makes Vietnamese ##Cà Phê## (coffee) special is the use of ##Sữa Đặc## (sweetened condensed milk), because fresh milk was not readily available at the time. The term ##Cà Phê Sữa## is a combination of the ##Cà Phê## (coffee) and ##Sữa Đặc## (sweetened condensed milk) to mean Vietnamese coffee. ##Cà Phê Sữa## (Vietnamese coffee) can be enjoyed hot or cold. ##Cà Phê Sữa## (Vietnamese coffee) is usually presented with the ##Sữa Đặc## (sweetened condensed milk) in a cup covered with a ##Phin Lọc## (Phin filter) that drips the ##Cà Phê## (coffee) over the ##Sữa Đặc## (sweetened condensed milk).";
```